install.md

```diff
@@ -153,13 +153,38 @@ This creates a file in the admin user's .ssh directory containing the secret. No
 
 ## Setup Server-side TLS/SSL 
 
-TLS must be enabled on the Harness Server and on all SDKs that communicate with it. That means at very least Python must be configured because the CLI uses the Python SDK.
+TLS can be toggled via HARNESS_SSL_ENABLED env variable (true|false). When set to 'true' it must be enabled on the Harness Server and on all SDKs that communicate with it. That means at very least Python must be configured because the CLI uses the Python SDK.
+
+    export HARNESS_SSL_ENABLED=${HARNESS_SSL_ENABLED:-false}
+TLS configuration can be configured at the file that HARNESS_SSL_CONFIG_PATH points to.
+
+    export HARNESS_SSL_CONFIG_PATH=${HARNESS_SSL_CONFIG_PATH:-conf/akka-ssl.conf}
+The content of the file is similar to the Akka SSL configuration:
```

Comment:
This path will never change why surface it as an env var? Also it will not exist in some future binary release. I really don't understand the need to look at .conf at runtime. The .conf should be set to override some things with env vars, but should never be references by env vars.

We really need to move away from config known only at compile time unless it does not change but we want to allow someone to change it and re-compile. All user changeable settings should not reference .conf files since in the future they will not be distributed in the binary dist.